reactapp/src/pages/SensorData.jsx

```diff
@@ -139,13 +213,31 @@ export default function SensorData() {
             modeState="application"
           />
 
+          <SensorComponent
+            sensorName="Mask notification content"
+            sensorDescription="Convert the notification messages into a irreversible code by applying a hash function"
+            stateField={applicationSensor.mask_notification}
+            field="mask_notification"
+            modeState="application"
+          />
+
           <SensorComponent
             sensorName="Mask touch text"
             sensorDescription="Swaps all alphanumeric characters by A, a, and 1"
             stateField={applicationSensor.mask_touch_text}
             field="mask_touch_text"
             modeState="application"
           />
+
+          <SensorComponent
+            sensorName="Text tracker"
+            sensorDescription="Log text on the screen. By default all information shown on the screen will be recorded."
```

Comment:
Could write "Log text displayed on the screen. By default, all information except password fields will be recorded."